packages/angular-ui-components/src/controllers/index.ts

```diff
@@ -1,3 +1,6 @@
+// Components
+export { ToasterControllerComponent } from './toaster/toaster-controller.component';
```

Comment:
Instead of doing this here. Create an index inside of `/toaster` with all the thing that are needed to export. And export it here.

Reply:
Will do :+1:

Reply:
Should I do `export * from './toaster/toaster-controller.component'` or `export { ToasterControllerComponent } from './toaster/toaster-controller.component';`?

Reply:
Checking the workaround of the angular libraries, they used to export all

![Screenshot 2022-08-30 at 15 42 24](https://user-images.githubusercontent.com/3854939/187467296-d2c2d989-bf9d-414f-a3ad-0b766fcb18d9.png)